packages/ListBox/src/components/Box/Box.styles.js

```diff
@@ -1,22 +1,26 @@
 import styled, { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
 
-const isInlineCSS = `
-  box-shadow: 0 0 0;
+const isInlineCSS = css`
+  border-top: 0;
   border-top-left-radius: 0;
   border-top-right-radius: 0;
-  border-top: 0;
+  box-shadow: 0 0 0;
 `;
 
 export const Box = styled.div(
-  ({ isInline, triggerWidth }) => css`
-    background-color: #ffffff;
-    border: 1px solid #d7d7d7;
-    border-radius: 3px;
-    box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.07), 0px 7px 17px rgba(0, 0, 0, 0.1);
+  ({ hasError, isInline, isReadOnly, triggerWidth }) => css`
+    background-color: ${tokens.color.white};
```

Comment:
thank you I think this has been like this even before we got a tokens packages :P